plugins/ohif/build.sh

```diff
@@ -53,7 +53,7 @@ rm -rf ${install_dir}
 mv ./Viewers/platform/viewer/dist ${install_dir}
 echo "Copied OHIF to ${install_dir}"
 
-rm -rf Viewers
```

Comment:
lets revert this file..

Reply:
thanks, it seems like this is reverted by commit from Andres below. If not, I can look into reverting this today.